recipes/pluginbase/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set name = "pluginbase" %}
+{% set version = "0.5" %}
+{% set sha256 = "b4f830242a078a4f44c978a84f3365bba4d008fdd71a591c71447f4df35354dd" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  host:
+    - python
+    - setuptools
+  run:
+    - python
+
```

Comment:
You need to add a `test` section with an `imports` subsection to test the import of the package.

Reply:
Okay, that is a good idea.